ui/components/Password.tsx

```diff
@@ -0,0 +1,22 @@
+'use client';
+import { Button } from '@/lib/Button';
+
+export default function Password() {
```

Comment:
wouldn't it be more idiomatic to do something like:

```
const [pass, setPass] = useState<string>("");
...
... password: pass
```

Reply:
Yes